### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://andriy-shymkiv.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/blocks/stars.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  height: 16px;
+  width: 16px;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star--active {
+  background-image: url(../images/star-active.svg);
```
Comment (taraselnik) [inline src/blocks/stars.css:17]: this one part we have to change accordingly html modifier. but you still able to look for the beter way.

(check the PATH'es to files )

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(../images/star-active.svg);
```

**Review by taraselnik — COMMENTED** [review]
my first review. I hope you'll enjoy it.  and thanks to making this review I found mistakes in my codE.
You've made a nice job IMHO.

File: src/blocks/stars.css (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/blocks/stars.css:17]: first i did it through html and after i understood that beter to do this from css

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+      <p class="card_code">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+        </div>
```
[the remark this answers is not in the document]
Comment (Andriy-Shymkiv) [reply src/index.html:29]: got it)

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,44 @@
...
+    <div data-qa="card" class="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card_logo"
+      >
+      <h2 class="card_name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card_code">Product code: 195434</p>
```
Comment (Olexandr6) [inline src/index.html:20]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 3rd point

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,44 @@
...
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__review">Reviews: 5</div>
```
Comment (Olexandr6) [inline src/index.html:30]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 12th point

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,44 @@
...
+      </div>
+
+      <div class="card__price">
+        <span class="card__price-title">Price</span>
+        <span class="card__price-amount">$2,199</span>
```
Comment (Olexandr6) [inline src/index.html:35]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 12th point

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,44 @@
...
+      </div>
+      <a
```
Comment (Olexandr6) [inline src/index.html:37]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 3rd point 
```suggestion
      </div>
      
      <a
```

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,44 @@
...
+        href="https://www.apple.com/ua/imac-24/"
+        data-qa="hover"
+        class="card__buy-button">Buy
+      </a>
```
Comment (Olexandr6) [inline src/index.html:41]: Pay attention to formating HTML!
```suggestion
      <a
          href="https://www.apple.com/ua/imac-24/"
          data-qa="hover"
          class="card__buy-button"
        >
          Buy
      </a>
```

File: src/normalize.css [hunk 8]
```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(./Roboto/Roboto-Medium.ttf);
+}
```
Comment (Olexandr6) [inline src/normalize.css:4]: Better to use CDN

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
### Pls pay attention to the `Readme.md`. Next time I will reject immediately
Why did you download all Roboto fonts manually to the project?
Here is the hint in the `Readme.md`, you should use just google fonts CDN👇

image

If you fix linking fonts, your solution will pass the tests

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**